src/client.ts

```diff
@@ -89,6 +90,7 @@ class DefaultClient implements Client {
 
 	constructor(config: Config) {
 		this.amplitudeInstance = createInstance();
+		this.amplitudeInstance.add(engagementPlugin());
```

Comment:
do we want to add it for the marketingClient as well ? https://github.com/balena-io-modules/analytics-client/blob/e125e54e758b1778190ca6b832acb41a33e6b42e/src/marketingClient.ts#L27

Reply:
Not at the moment